test/ocl_to_spirv_for_opencl_cpp_with_unnamed_struct.ll

```diff
@@ -0,0 +1,17 @@
+;; This test checks if builtins in LLVM IR generated from OpenCL_CPP sources
+;; including unnamed structs are correctly translated to SPIR-V builtin variables.
```

Comment:
Wait, in SPIR-V builtin variables has a concrete meaning, see https://registry.khronos.org/SPIR-V/specs/unified1/SPIRV.html#BuiltIn . I don't see any builtins here, so likely to be a misuse of the word

Reply:
Thanks for spotting this!